docs/API.md

```diff
@@ -955,42 +955,38 @@ __Parameters__
 | Param  |  Type | Description  |
 |---|---|---|
 | `bucketName`  | _string_  | Name of the bucket |
-| `objectPrefix` | _string_ | Prefix of objects in the bucket with which to filter permissions off of. Use `''` for entire bucket. |
-| `callback(err, policy)`  | _function_  | Callback function is called with non `null` err value in case of error. `policy` will be the string representation of the bucket policy (`minio.Policy.NONE`, `minio.Policy.READONLY`, `minio.Policy.WRITEONLY`, or `minio.Policy.READWRITE`). If no callback is passed, a `Promise` is returned. |
+| `callback(err, policy)`  | _function_  | Callback function is called with non `null` err value in case of error. `policy` will be the string representation of the bucket policy file. If no callback is passed, a `Promise` is returned. |
 
 
 ```js
 // Retrieve bucket policy of 'my-bucketname' that applies to all objects that
 // start with 'img-'.
-minioClient.getBucketPolicy('my-bucketname', 'img-', function(err, policy) {
+minioClient.getBucketPolicy('my-bucketname', function(err, policy) {
```

Comment:
This example doesn't make sense anymore since `prefix` is not required to be provided anymore. If we want to keep the example, then we need to modify the comment/explanation.